conga-aem-definitions/src/main/templates/aem-dispatcher-ams/conf.d/available_vhosts/tenant.vhost.partials.hbs

```diff
@@ -45,9 +45,9 @@ CustomLog logs/vhost_{{tenant}}_access.log {{httpd.logging.accessLogFormat}}
 {{~#block "generalSettings"}}
 # Use a document root that matches the one in conf.dispatcher.d/default.farm
 DocumentRoot "${PUBLISH_DOCROOT}"
-# Add header breadcrumbs for help in troubleshooting
 <IfModule mod_headers.c>
-  Header add X-Vhost "publish"
+  # Add header breadcrumbs for help in troubleshooting
+  Header add X-Vhost "{{httpd.serverName}}"
```

Comment:
this should also be encapsulated in `{{#httpdCloudManagerConditional}}` (as it is done for AEMaaCS) - the conditional expression is used above in this file for determining the server name as well.

Reply:
done